src/containers/Tablet/TabletControls/TabletControls.tsx

```diff
@@ -17,18 +17,19 @@ type VisibleDialogType = EVisibleDialogType | null;
 
 interface TabletControlsProps {
     tablet: TTabletStateInfo;
+    fetchData: VoidFunction;
 }
 
-export const TabletControls = ({tablet}: TabletControlsProps) => {
+export const TabletControls = ({tablet, fetchData}: TabletControlsProps) => {
     const {TabletId, HiveId} = tablet;
 
     const [isDialogVisible, setIsDialogVisible] = useState(false);
     const [visibleDialogType, setVisibleDialogType] = useState<VisibleDialogType>(null);
-    const [isTabletActionsDisabled, setIsTabletActionsDisabled] = useState(false);
+    const [isTabletActionLoading, setIsTabletActionLoading] = useState(false);
```

Comment:
Here we control state internally, because it is affected only by actions, triggered by `TabletControls`, that are defined in `TabletControls` as well.
For regular fetches (external) loading state should not be set (still it resets to true when new data is received) . 

Probably it all can be changed with component logic redesign, but I don’t want to do it for the current issue

Reply:
Ok, let's stick to this, but remove reseting when new data is received (see next comment).